place_tracker/lib/place_map.dart

```diff
@@ -469,15 +469,15 @@ class _AddPlaceButtonBar extends StatelessWidget {
           alignment: MainAxisAlignment.center,
           children: [
             ElevatedButton(
-              style: ElevatedButton.styleFrom(primary: Colors.blue),
+              style: ElevatedButton.styleFrom(foregroundColor: Colors.blue),
```

Comment:
I wonder what's the best way to communicate these parameter simplifications. Obviously, folks will run into them on their own, so, maybe nothing? But, I dunno. Could warrant _something_.